pkg/compute/executor.go

```diff
@@ -216,7 +221,13 @@ func (e *BaseExecutor) Start(ctx context.Context, execution *models.Execution) (
 		return
 	}
 
-	args, cleanup, err := PrepareRunArguments(ctx, e.Storages, execution, resultFolder)
+	executionStorage := filepath.Join(e.storageDirectory, fmt.Sprintf("%s/%s", execution.JobID, execution.ID))
```

Comment:
Why not just `filepath.Join(e.StorageDirectory, execution.JobID, execution.ID)` ?